packages/EasyTest/src/Traits/MessengerAssertionsTrait.php

```diff
@@ -266,11 +266,20 @@ private static function runMessengerWorker(
         foreach ($transport->getRejected() as $envelope) {
             /** @var \Symfony\Component\Messenger\Stamp\ErrorDetailsStamp $errorDetailsStamp */
             foreach ($envelope->all(ErrorDetailsStamp::class) as $errorDetailsStamp) {
-                foreach ($expectedExceptions as $key => $expectedExceptionPair) {
+                foreach ($expectedExceptions as $key => $expectedExceptionPairOrClass) {
+                    // if $expectedExceptionPairOrClass is a string = exceptionClass
+                    if ($expectedExceptionPairOrClass === $errorDetailsStamp->getExceptionClass() &&
```

Comment:
It looks like we can combine these two `if` into a single `if`. Wdyt?

Reply:
Sure. But it's split for readable purposes. It's more clear.

Reply:
I got it.